glommio/src/io/dma_file_stream.rs

```diff
@@ -988,14 +1007,13 @@ impl DmaStreamWriter {
             sync_on_close: builder.sync_on_close,
             current_buffer: None,
             waker: None,
-            pending: AHashMap::new(),
+            flushes: VecDeque::with_capacity(builder.write_behind * 2),
```

Comment:
why would we need the queue to be 2 * write_behind ?
In which situations do we expect to have more flushes than the number of write behind ?

Reply:
Out-of-order completions will still be present. In the worst case you have the oldest flush pending, then `wb - 1` completions, allowing for `wb - 1` flushes to further be added to the queue. To keep it simple went with `wb * 2` for capacity.

Reply:
that makes sense.